omnibus/files/private-chef-cookbooks/chef-ha-native/attributes/default.rb

```diff
@@ -0,0 +1,25 @@
+#######
+# Bookshelf Sender
+#######
+default['private_chef']['bookshelf-sender']['enable'] = true
+default['private_chef']['bookshelf-sender']['ha'] = true
+default['private_chef']['bookshelf-sender']['dir'] = "/var/opt/opscode/bookshelf-sender"
+default['private_chef']['bookshelf-sender']['log_directory'] = "/var/log/opscode/bookshelf-sender"
+default['private_chef']['bookshelf-sender']['log_rotation']['file_maxbytes'] = 104857600
+default['private_chef']['bookshelf-sender']['log_rotation']['num_to_keep'] = 10
+default['private_chef']['bookshelf-sender']['remote_host'] = nil
+default['private_chef']['bookshelf-sender']['remote_port'] = 11873
+default['private_chef']['bookshelf-sender']['remote_username'] = 'opscode'
+default['private_chef']['bookshelf-sender']['remote_password'] = 'opscode'
+
+#######
+# Bookshelf Receiver
+#######
+default['private_chef']['bookshelf-receiver']['enable'] = true
+default['private_chef']['bookshelf-sender']['ha'] = true
```

Comment:
These should actually be false.  We'll need to handle these services a bit differently.